File: lib/templates/sqlite/flask/server/routes/push.py
from flask import Blueprint, request, jsonify
import db

push_bp = Blueprint('push', __name__)

@push_bp.route('/api/push', methods=['POST'])
def push():
    data = request.get_json()
    conn = db.get_db()
    conn.execute('INSERT INTO items (name, message) VALUES (?, ?)', [data['name'], data['message']])
    conn.commit()
    conn.close()
    return jsonify({'status': 'ok'})

File: lib/templates/sqlite/server/src/routes/push.ts
import { Router } from 'express'
import db from '../db'

const router = Router()

router.post('/', (req, res) => {
  const { name, message } = req.body
  db.prepare('INSERT INTO items (name, message) VALUES (?, ?)').run(name, message)
  res.json({ status: 'ok' })
})

export default router
